apps/web/src/views/Info/components/CurrencyLogo/index.tsx

```diff
@@ -28,7 +28,22 @@ export const CurrencyLogo: React.FC<
     return getTokenLogoURL(new Token(multiChainId[chainName], address as Address, 18, ''))
   }, [address, chainName])
 
-  const imagePath = chainName === 'BSC' ? '' : `${chainName?.toLowerCase()}/`
+  let imagePath = chainName?.toLowerCase()
+  switch (imagePath) {
+    case 'bsc':
+      imagePath = ''
+      break
+    case 'arb':
+      imagePath = 'arbitrum/'
+      break
+    case 'polygon_zkevm':
+      imagePath = 'polygon-zkevm/'
+      break
+    default:
+      imagePath += '/'
+      break
+  }
```

Comment:
Avoid duplicate code.
Try use `getImageUrlFromToken`

Reply:
fixed

Reply:
You should not remove the image for `trustwallet`.
We need both, trustwallet image and ourself image url.